src/Rector/AbstractRector.php

```diff
@@ -305,6 +305,11 @@ protected function isName(Node $node, string $name): bool
         return $this->nodeNameResolver->isName($node, $name);
     }
 
+    protected function isNameExactly(Node $node, string $name): bool
+    {
+        return $this->nodeNameResolver->isNameExactly($node, $name);
+    }
+
```

Comment:
This method is not needed. Better use `nodeNameResolver` directly, to avoid duplicating whole API of `nodeNameResolver` here